README.md

```diff
@@ -1,2 +1,54 @@
 # cronbeat
```

Comment:
@jetbrains-junie update readme from main and extend it with project setup and development information like you did here